packages/markdown/src/index.test.ts

```diff
@@ -0,0 +1,136 @@
+import { assert } from 'chai';
+
+import * as markdown from './index.js';
+
+async function testMarkdown(
+  original: string,
+  expected: string,
+  options: { inline?: boolean } = {},
+) {
+  const actual = await markdown.markdownToHtml(original, options);
+  assert.equal(actual.toString().trim(), expected);
+}
+
+describe('Markdown processing', () => {
```

Comment:
There seems to be a lot of overlap, if not complete duplication, with the tests in `apps/prairielearn/src/lib/markdown.test.ts`. Can we deduplicate?

Reply:
Deduplicated in f82e6cf49.